pandora-client-web/src/components/chatroomSelect/chatroomSelect.tsx

```diff
@@ -170,7 +170,21 @@ function RoomEntry({ roomInfo }: {
 						title={ hasPassword ? 'Protected room' : 'Open room' }
 						alt={ hasPassword ? 'Protected room' : 'Open room' } />
 				</div>
-				<div className='entry'>{ `${name} (${users}/${maxUsers})` }</div>
+				<div className='entry'>
+					{ name }
+					{ ' ' }
+					(
+					{ ' ' }
+					{ onlineCharacters }
+					{ ' ' }
+					<span className='offlineCount'>(+{ totalCharacters - onlineCharacters })</span>
+					{ ' ' }
+					/
+					{ ' ' }
+					{ maxUsers }
+					{ ' ' }
+					)
```

Comment:
Not sure if this is more readable or even work but I got this idea:
(feel free to close this without change)

```suggestion
					{ `${name} ( ${onlineCharacters} ` }
					<span className='offlineCount'>(+{ totalCharacters - onlineCharacters })</span>
					{ ` / ${maxUsers} )` }
```

Reply:
Yes, this seems to work, too. It is Jomshir's code, so I will ask if that change is okay. It surely looks more readable to me.

Reply:
The change looks good to me~